### ISSUE TITLE
why switch to `index.js`?

### ISSUE THREAD

**Issue by cklanac** [Thinkful-Ed/backend-template#1]
@oampo, just curious... why switch from `server.js` to `index.js`?

I know it's not terribly important but I'm torn between the two names. 

- `server.js` is a standard in npm since `npm start` runs `node server.js` by default
- `index.js` is a standard cause well, it's `index.js` and it seems like it should be the default. And `require()` automatically loads the index of a folder but that doesn't seem to apply for this situation.

Of course opt for `app.js` 'cause it is used in Express Generator or `main.js` 'cause its' a classic.

**Comment by oampo** [comment]
Yeah, there are competing standards here - Node prefers `index.js` (`node dirname` runs `node dirname/index.js`), NPM prefers `server.js` for a default `npm start`.  I figured that maybe consistency with React is good, so `index.js` is our entry point everywhere.  But I don't care too much - if you have a strong preference then feel free to change it.

**Comment by cklanac** [comment]
I'm leaning towards `server.js` in the short term just to avoid updating all the repos, glitch and curric references. Let's discuss it in the context of updating node to use the learnings from bellyful. I'd like to discuss what we keep, what do eject, and what we need to simplify for pedagogical reasons.
